packages/vite-plugin-checker/src/checkers/vueTsc/prepareVueTsc.ts

```diff
@@ -8,7 +8,7 @@ const _require = createRequire(import.meta.url)
 const _filename = fileURLToPath(import.meta.url)
 const _dirname = dirname(_filename)
 
-const proxyPath = _require.resolve('vue-tsc/out/proxy')
+const proxyPath = _require.resolve('vue-tsc/out/index')
```

Comment:
I think we should still be compatible with `vue-tsc/out/proxy` for a few days until we remove it in a breaking version in the future. So detecting these two files and deciding which to patch will be great at this time.

Reply:
@johnsoncodehk shimed it in v10.16 See https://github.com/johnsoncodehk/volar/commit/5d7581e396e7811ab51fad9494e25c56771e5a98
Do you still think it's worth handling both?

Reply:
Yes, some users might still install the latest vite-plugin-checker and old version of vue-tsc in the future. Since we can be compatible with both at a very low cost, we can support the old version for longer bug fixes and new features and drop it in the future breaking change version.

Reply:
@fi3ework 👍 done! It now tries to read /out/index, then falls back to /out/proxy if not found
I tested locally with vue-tsc v1.0.13 (/out/proxy), v1.0.14 (/out/index) and v1.0.16 (/out/proxy shimed)